plugins/browser-plugin-optimizely-x/src/index.ts

```diff
@@ -33,8 +33,8 @@ import { OptimizelyxSummary } from './contexts';
 
 declare global {
   interface Window {
-    optimizely: {
-      get: (property: string) => { [key: string]: any };
+    optimizely?: {
```

Comment:
Just to be sure – do you see any potential of this being a breaking change? When I ran `rush build`, it first gave me this error:

```
==[ @snowplow/browser-plugin-optimizely-x ]======================[ 13 of 34 ]==                   
[!] (plugin Typescript) TS2687: All declarations of 'optimizely' must have identical modifiers.   
src/index.ts (36:5)                                                                               
                                                                                                  
36     optimizely?: {                                                                             
       ~~~~~~~~~~                                                                                 
                                                                                                  
                                                                                                  
Returned error code: 1                                                                            
"@snowplow/browser-plugin-optimizely-x" failed to build.  
```

After running `rush clean`, it worked. This made me think that maybe in case users have defined their own global Window type with optimizely, it could be breaking for their build to change the type here? Maybe this is not an issue at all or it's super unlikely to be an issue, so I don't think it needs to be a blocker, but I thought I'd check.

Reply:
Hmm, I'm just thinking maybe the error was a conflict in the type definition with the older optimizely plugin? It [defines the optimizely type here](https://github.com/snowplow/snowplow-javascript-tracker/blob/master/plugins/browser-plugin-optimizely/src/index.ts#L44).

Reply:
I also had this issue, but the "conflicting definition" was the same plugin's `dist` output, so I figured it was some kind of bug in our tsconfig include paths or something, but you could be right.

The Optimizely SDK itself doesn't seem to declare any type at the global level so I'm not sure what the best approach would be. I don't mind backing out the type change and just keeping the defensive code?

Reply:
Yeah, I think you're right that the conflict was with the `dist` output of the same plugin, I just ran into it again and removing the `dist` folder and building again solved it.

But it seems reasonable to keep the defensive code but undo the type change if we're not 100% sure about what the type change might do... Most likely won't be a problem, but I'm not super confident about it.